### ISSUE TITLE
Execute Darknet Command from External Python Script

### ISSUE THREAD

**Issue by amar-mustaqim** [AlexeyAB/darknet#2991]
Hi. Is it possible to execute darknet command from an external python script? I try to use subprocess module but it doesnt work. This is my code: 

import subprocess
subprocess.call("C:/Users/Admin/Desktop/CSP650/darknet-master (1)/darknet-master/build/darknet/x64/darknet.exe detector demo data/obj.data yolo-obj.cfg backup/yolo-obj_last.weights -c 0")

**Comment by Arsiirasheed** [comment]
yes its possible  i having been doing it on google colab python colab notebook . make sure your  obj.data file exists or try giving absolute path

**Comment by LukeAI** [comment]
If you are calling darknet from a different location but giving relative paths then it will fail.
